### PR TITLE
Exclude Jep425Tests_testVirtualThread

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Exclude `Jep425Tests_testVirtualThread`

https://github.com/eclipse-openj9/openj9/issues/15650
https://github.com/eclipse-openj9/openj9/issues/15651

FYI @tajila 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: test/functional/Java19andUp/playlist.xml [hunk 1]
```diff
@@ -23,6 +23,12 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>Jep425Tests_testVirtualThread</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15650</comment>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15651</comment>
```
Comment (llxia) [inline test/functional/Java19andUp/playlist.xml:29]: This will not pass the TKG check. Only one comment is expected.

**Comment by pshipton** [comment]
@tajila is this to be excluded permanently or temporarily?

**Comment by tajila** [comment]
> @tajila is this to be excluded permanently or temporarily?

temporarily

File: test/functional/Java19andUp/playlist.xml (hunk 1 above)
Comment (JasonFengJ9) [reply test/functional/Java19andUp/playlist.xml]: Yeah, removed the second one, and verified locally.

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/1212/

**Pull request merged**